src/common/vordr.ts

```diff
@@ -34,19 +34,20 @@ type CheckWithVordrInput = {
   content?: string;
 };
 
-type CheckWithVordrContext = {
-  userId: string;
-  con: DataSource | EntityManager;
-  req?: Pick<FastifyRequest, 'ip'>;
-};
-
 export const checkWithVordr = async (
   { id, type, content }: CheckWithVordrInput,
-  { userId, con, req }: CheckWithVordrContext,
+  ctx: Context,
 ): Promise<boolean> => {
-  if (req && validateVordrIPs(req.ip)) {
+  const { userId } = ctx;
```

Comment:
also I don't think it always comes from ctx, sometimes its used in different scope